deployment/media-insights-stack.yaml

```diff
@@ -89,9 +89,9 @@ Parameters:
   ApiIpList:
     Type: String
     Description:
-      CIDR block string.  Indicates which internet addresses can access Elastichsearch.
+      CIDR block string.  Indicates which internet addresses can access the MIE APIs.
```

Comment:
Currently, this only applied to Elasticsearch and not the MIE APIs.  However, I think this line is going to get taken out by Elasticsearch changes from @brandold anyhow.